docs/semgrep-app/editor.md

```diff
@@ -0,0 +1,205 @@
+
+To quickly learn Semgrep patterns and syntax, explore the Editor’s library of rules from the **public [Rule Registry](https://semgrep.dev/r)**. Rules from the Registry can detect OWASP vulnerabilities, best practice violations, and security issues for a wide variety of languages and frameworks. Semgrep Editor enables you to **adapt these rules** for your own organization’s use by [forking](#jumpstart-rule-writing-using-existing-rules) them.
+
+The Editor is free to use on all tiers, but saving a rule for **private use** is a **Team/Enterprise tier feature**.
+
+## Accessing Semgrep Editor
+
+1. Sign in to your [Semgrep App account](https://semgrep.dev/login).
+2. Click **Editor (beta)** from the **App sidebar**. The **splash page** and **library pane** appear.
+3. Do any of the following steps:
+    1. To create a new rule, click on the **(+) plus sign** or **Create new rule** button.
+    2. To open any rule you’ve recently edited, select it from the **Recent** list.
+    3. To view a sample rule, select it from the **Examples** list. The rule renders within the Editor.
+    4. To start a tutorial or read the docs, select it from the **Learn** list. This navigates you away from the Editor.
+
+## Viewing rules
+
+![Screenshot of the Editor's main view composed of three panes and top menu](../img/editor-panes.png)
+
+The editor is composed of three panes and a top menu.
+
+<dl>
+    <dt>Library</dt>
+    <dd>View and open various rules through the library. It contains two top-level folders:
+        <ul>
+            <li>Your organization’s rules</li>
+            <li>Publicly-available Semgrep Registry rules from the <code>semgrep-rules</code> repository</li>
+        </ul>
+        <strong>Private rules</strong> are denoted by a <strong>lock</strong> icon overlaid over the file icon.
+    </dd>
+    <dt>YAML editor</dt>
+    <dd>Enter your rule’s YAML schema in this pane.</dd>
+    <dt>Sample code window</dt>
+    <dd>Enter your rule’s test code in this pane and click the <strong>Run button</strong> to verify that the rule runs as intended. A <strong>matches panel</strong> appears after Semgrep is run to display matches and tests.</dd>
+    <dt>Top menu</dt>
+    <dd>Save, share, and add your rule to the Rule Board through this menu.</dd>
+</dl>
+
+To **resize the panes**, position your mouse over the borders and click-drag to the desired width. The library pane can be completely hidden to give more space for the editing panes.
+
+Semgrep Registry rules are initially grouped by **directory**. Most of these directories correspond to languages. The Library can also be grouped by **rulesets**, which are rules arranged by category, such as security, best practices, and frameworks.
+
+To group by ruleset, right click on the empty space on the **official registry** entry and select **Group by ruleset**.
+
+![editor-groupbyruleset.png](../img/editor-groupbyruleset.png)
```

Comment:
For this screenshot and a few of the others I think it would help to add a light border, since the screenshot’s background and the page’s background are both white (in non-dark mode). I’m not sure the easiest way to do this. It could be in the screenshot directly, or you could replace the Markdown image with the html tag for the image (`<img alt="editor-groupbyruleset.png" src= bla bla bla`) and add an inline style directly or add a CSS class that then attaches something like `border: 3px solid #eee;` or something like that.

Reply:
I agree! And I think some of the screenshots should also be smaller... for now I'll try to add a css value for width.